sdk/resourcemanager/azure-resourcemanager-compute/src/test/java/com/azure/resourcemanager/compute/implementation/VirtualMachineUpdateTests.java

```diff
@@ -37,7 +38,9 @@ protected void cleanUpResources() {
     }
 
     @Test
+    @DoNotRecord(skipInPlayback = true)
     public void testVirtualMachineUpdate() {
+        // Management Long running operation is Failed or Cancelled
```

Comment:
cause?

Reply:
Regardless of `live` mode or `record` mode, `Long running operation is Failed or Cancelled` Exception occurs. The detail information following as:
[test.log](https://github.com/user-attachments/files/16689008/test.log)
[compute-test.log](https://github.com/user-attachments/files/16689016/compute-test.log)

Reply:
`testVirtualMachineUpdate` is the basic test case. It MUST pass.

Reply:
@v-hongli1 Please remove the log link in the comment and investigate the LRO failure cause.

Reply:
Same comment as Xiaofei. What we need from you is to either fix the issue (e.g. if a simple update to different region/sku), or at least point to place in code where it fails. Do not paste the log.

Reply:
WIP https://github.com/Azure/azure-sdk-for-java/pull/41596

Reply:
Fixed it with WIP https://github.com/Azure/azure-sdk-for-java/pull/41596